build.gradle

```diff
@@ -4,6 +4,9 @@ buildscript {
     apply from: 'buildsystem/constants.gradle'
     apply from: 'buildsystem/helpers.gradle'
     apply from: 'buildsystem/dependencies.gradle'
+    ext {
+        versions.kotlin = '1.3.61'
```

Comment:
This one is not necessary, as it the value is already available in `buildsystem/dependencies.gradle` and is actually pointing to a newer release.

Reply:
Fixed in cb30a2d8773d8303c1fb68fcd7dc1bd13b0571ce